src/Acme/Bundle/EnrichBundle/Form/Type/TranslatableCategoryType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Acme\Bundle\EnrichBundle\Form\Type;
+
+use Symfony\Component\Form\FormBuilderInterface;
+
+use Pim\Bundle\EnrichBundle\Form\Type\CategoryType as BaseCategoryType;
+
+/**
+ * Type for category properties
+ */
+class CategoryType extends BaseCategoryType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        parent::buildForm($builder, $options);
+        
+        $builder->add(
+            'description',
+            'pim_translatable_field',
+            [
+                'field'             => 'description',
+                'translation_class' => $this->translationdataClass,
```

Comment:
Nice catch on this one !
I will propose a fix on master and do a PR on the doc (master) too.